helper/testhelpers/testhelpers.go

```diff
@@ -974,3 +976,13 @@ func SetupLoginMFATOTP(t testing.T, client *api.Client) (*api.Client, string, st
 	SetupMFALoginEnforcement(t, client, enforcementConfig)
 	return entityClient, entityID, methodID
 }
+
+func SkipUnlessEnvVarsSet(t testing.T, envVars []string) {
+	t.Helper()
+
+	for _, i := range envVars {
+		if os.Getenv(i) == "" {
```

Comment:
Perhaps you should rather use:  `os.LookupEnv()`, which returns a string value and a bool that indicates if the env var was set, i.e.

```
  if _, isSet := os.LookupEnv(i); !isSet {
    ....
  }
```

Reply:
This was a consideration, but the reason that I didn't go with it is that it doesn't provide me useful choices beyond complicating handling the return with no change to the outcome I was seeking. How does that sit with you?